LAMPStack_Install/recipes/php.rb

```diff
@@ -0,0 +1,42 @@
+package "php5" do
+  action :install
+end
+
+package "php-pear" do
+  action :install
+end
+
+package "php5-cgi" do
+  action :install
+end
+
+package "php5-dev" do
```

Comment:
instead of four package resources, use only one with comma-separated list of all package

Reply:
done

Reply:
this is not done yet?